apps/velo-external-db/test/e2e/app_data.e2e.spec.ts

```diff
@@ -307,7 +307,7 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
     })
 
     describe('error handling', () => {
-        test('insert api with duplicate _id should fail with WDE0074, 409', async() => {
+        test.skip('insert api with duplicate _id should fail with WDE0074, 409', async() => {
```

Comment:
?

Reply:
These tests will be skipped, It will be changed after https://github.com/wix/velo-external-db/pull/421 will be merged to https://github.com/wix/velo-external-db/pull/352, https://github.com/wix/velo-external-db/pull/421 changed these tests and added new supported tests.